shortReads_assembly.wdl

```diff
@@ -214,6 +217,7 @@ task finish_asm {
         cat ~{scaffold} | sed ~{sed} > ~{prefix}_scaffolds.fna
         cat ~{covstats} | sed ~{sed} > ~{prefix}_covstats.txt
         cat ~{agp} | sed ~{sed} > ~{prefix}_assembly.agp
+        cat ~{bbcms_fastq} | sed ~{sed} > ~{prefix}_bbcms.fastq.gz
```

Comment:
This file doesn't need to go through a sed command Ie the headers don't need to change, the file itself just needs to be changed.